WebRoot/skins/_base/base4/skin.css

```diff
@@ -1066,3 +1066,41 @@ div.ZmHtmlEditor .TxtAreaDrag {
 .ZmConvDoublePaneView .Row-selected .ZmMsgListDate {
     background-color: @ListItemBgColor-selected@ !important;
 }
+
+/*Circular Progress bar styling*/
+svg.progress-circle {
+	@FullWidth@
+}
+
+svg.progress-circle text {
+	fill: @TxtC@;
+}
+
+svg.progress-circle circle.bg {
+	stroke: #e5e5e5;
```

Comment:
Added properties for all possible variable as discussed:

https://github.com/Zimbra/zm-web-client/pull/63/commits/e1a07e87f8935a96ddd70132276b5355ec73786a